src/06-products/products.apib

```diff
@@ -24,3 +24,5 @@ Get a list of products.
                         + name: `cookies` (string)
                         + description: `dark chocolate` (string)
                 + external_id: `123` (string)
+                + purchase_price (Money, optional)
```

Comment:
I believe this can be `null` but not `optional`

Reply:
Okay what should we do? I am a bit lost.